### PR TITLE
refactor(Switch): match Figma and utilize Tailwind

### PR DESCRIPTION
**Pull request by justinhough**
**Related Issue:** #2461 

## Summary
Updates calcite-switch hover state on idle and active to grow inward as shown in Figma design.

- Changed `box-shadow` to use inset rather than the previous `shadow-outline-active`

### CODE REVIEW

**Review by bstifle — APPROVED** [review]
@justinhough really nice work, possibly perfect.

**Comment by bstifle** [comment]
@justinhough looks like i missed a detail.

image


let's get that ui-brand-hover 1px border in there when active

**Comment by justinhough** [comment]
@bstifle Got it. The update has been made.

**Comment by justinhough** [comment]
@julio8a The build failed, but not sure what it failed on.

**Comment by jcfranco** [comment]
@justinhough The build failed because of an unstable test. You can see this by going to the details in the Travis CI check and then clicking on the link to the build output. 
Screen Shot 2021-08-02 at 2 53 30 PM

Could you merge `master` (the test is skipped now) and have the build run again?

**Comment by bstifle** [comment]
lookin good!

**Comment by justinhough** [comment]
> @justinhough The build failed because of an unstable test. You can see this by going to the details in the Travis CI check and then clicking on the link to the build output.
> Screen Shot 2021-08-02 at 2 53 30 PM
> 
> Could you merge `master` (the test is skipped now) and have the build run again?

Just put in the merge. Thanks.

**Comment by julio8a** [comment]
When it fails, I usually merge master and push it up or push up an empty commit and that usually fixes it.

File: src/components/calcite-switch/calcite-switch.scss [hunk 1]
```diff
@@ -84,14 +84,15 @@
 :host(:hover),
 :host(:focus) {
   .handle {
-    @apply border-color-brand shadow-outline-active;
+    @apply border-color-brand-hover;
+    box-shadow: inset 0 0 0 1px var(--calcite-ui-brand-hover);
```
Comment (caripizza) [inline src/components/calcite-switch/calcite-switch.scss:88]: I think these two should be `brand` (not `brand-hover`)

File: src/components/calcite-switch/calcite-switch.scss [hunk 2]
```diff
@@ -84,14 +84,15 @@
...
     @apply right-auto;
```
Comment (caripizza) [inline src/components/calcite-switch/calcite-switch.scss:89]: It also looks like `@apply right-auto;` is adding a wobble effect to the handle on :hover when dir is "rtl". Can you try removing to see if it fixes that issue?

**Comment by caripizza** [comment]
Thanks @justinhough, the colors LGTM! I think the last remaining item is to add the heights for each scale's `.container` class - 12px/16px/24px. (This was the vertical alignment issue I mentioned in our Zoom chat today.) 

Hopefully it will resolve the issue with the switch not being vertically aligned in the middle of the div when in :focus:
image

```scss
/* ex: height set per scale */
:host([scale="s"]) {
  .container {
    @apply h-3;
  }
  .track {
    ...
}
:host([scale="m"]) {
  .container {
    @apply h-4;
  }
  .track {
  ...
}

:host([scale="l"]) {
  .container {
    @apply h-6;
  }
  .track {
    ...
```

**Comment by justinhough** [comment]
@caripizza Added in the update for the container heights and it looks to have fixed that vertical align issue that you mentioned. Thanks.

**Review by caripizza — APPROVED** [review]
thanks @justinhough! 👍

**Pull request merged**